ci/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG TENDERMINT_VERSION=0.34.11
 ARG GAIA_VERSION=5.0.5
-ARG RUST_VERSION=1.57
+ARG RUST_VERSION=1.59
```

Comment:
Do we want to bump to 1.60 right away?

Reply:
https://github.com/informalsystems/basecoin-rs/pull/44/commits/e234d91a7708d71d98e72d39b0f5cf05b25ec339